src/HTTP/Response/rendering/ResponseRenderingStrategy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace ILIAS\HTTP\Response\rendering;
+
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Interface ResponseRenderingStrategy
+ *
+ * @author  Nicolas Schäfli <ns@studer-raimann.ch>
+ */
+interface ResponseRenderingStrategy {
```

Comment:
@klees 
That is a good point, I will update the names within the package.

They are strategies because of the strategy pattern used within the service. The senders are representing the strategies, the sender interface represent the abstraction and the HTTPServices class represents the context. I adopted the naming scheme of the factories.